tests/backup/backup_test.go

```diff
@@ -292,40 +298,63 @@ func DeleteBackupLocation(name string, orgID string) {
 	})
 }
 
-func CreateCluster(name string, cloudCred string, orgID string) {
+func CreateSourceAndDestClusters(cloudCred string, orgID string) {
+	// TODO: Add support for adding multiple clusters from
+	// comma separated list of kubeconfig files
+	kubeconfigs := os.Getenv("KUBECONFIGS")
+	Expect(kubeconfigs).NotTo(Equal(""),
+		"KUBECONFIGS Environment variable should not be empty")
 
-	Step(fmt.Sprintf("Create cluster [%s] in org [%s]", name, orgID), func() {
-		backupDriver := Inst().Backup
+	kubeconfigList := strings.Split(kubeconfigs, ",")
+	// Validate user has provided at least 2 kubeconfigs for source and destination cluster
+	Expect(len(kubeconfigList)).Should(BeNumerically(">=", 2), "At least minimum two kubeconfigs required")
 
-		// TODO: Add support for adding multiple clusters from
-		// comma separated list of kubeconfig files
-		kubeconfigPath := os.Getenv("PXBACKUP_KUBECONFIG")
-		Expect(kubeconfigPath).NotTo(Equal(""),
-			"PXBACKUP_KUBECONFIG Environment variable should not be empty")
+	err := dumpKubeConfigs(ConfigMapName, kubeconfigList)
+	Expect(err).NotTo(HaveOccurred(),
+		fmt.Sprintf("Failed to get kubeconfigs [%v] from configmap [%s]", kubeconfigList, ConfigMapName))
 
-		kubeconfigRaw, err := ioutil.ReadFile(kubeconfigPath)
+	// Register source cluster with backup driver
+	Step(fmt.Sprintf("Create cluster [%s] in org [%s]", SourceClusterName, orgID), func() {
+		srcClusterKubeConfig := fmt.Sprintf("%s/%s", KubeconfigDirectory, kubeconfigList[0])
+		_, err = Inst().Backup.CreateNewCluster(SourceClusterName, cloudCred, srcClusterKubeConfig, orgID)
```

Comment:
create a `ClusterCreateRequest` there then call `CreateCluster`, otherwise remove `CreateCluster` and stick to use only `CreateNewCluster ` cause I don't see `CreateCluster` being called anywhere else

Reply:
CreateCluster is used by backup-integration suite

Reply:
@thiguetta / @raviprasadsn removed `CreateNewCluster()` API call altogather.